<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Student_Registration</class>
 <widget class="QWidget" name="Student_Registration">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1036</width>
    <height>685</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>16</pointsize>
    <weight>75</weight>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Student Registration</string>
  </property>
  <property name="windowIcon">
   <iconset resource="img.qrc">
    <normaloff>:/new/img/Img/logo.jpg</normaloff>:/new/img/Img/logo.jpg</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(148, 255, 98);</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>441</width>
     <height>521</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Franklin Gothic Heavy</family>
     <pointsize>20</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 20pt &quot;Franklin Gothic Heavy&quot;;</string>
   </property>
   <property name="title">
    <string>Student Details</string>
   </property>
   <widget class="QPushButton" name="pushButton_Submit">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>460</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(55, 167, 81);
border-radius:20%;


font: 75 16pt &quot;MS Shell Dlg 2&quot;;
font: 75 16pt &quot;Rockwell&quot;;</string>
    </property>
    <property name="text">
     <string>Submit</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Reset">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>460</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(55, 167, 81);
border-radius:20%;


font: 75 16pt &quot;MS Shell Dlg 2&quot;;
font: 75 16pt &quot;Rockwell&quot;;</string>
    </property>
    <property name="text">
     <string>Reset</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>131</width>
      <height>34</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 16pt &quot;Trebuchet MS&quot;;</string>
    </property>
    <property name="text">
     <string>Last Name :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>400</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 16pt &quot;Trebuchet MS&quot;;</string>
    </property>
    <property name="text">
     <string>  Course :</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_course">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>390</y>
      <width>261</width>
      <height>41</height>
     </rect>
    </property>
    <property name="currentText">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 16pt &quot;Trebuchet MS&quot;;</string>
    </property>
    <property name="text">
     <string>Roll_No :</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_rno">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>160</y>
      <width>261</width>
      <height>41</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Enter your rollno</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>340</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 16pt &quot;Trebuchet MS&quot;;</string>
    </property>
    <property name="text">
     <string>Phone_Number :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>141</width>
      <height>35</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 16pt &quot;Trebuchet MS&quot;;</string>
    </property>
    <property name="text">
     <string>First Name :</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_fname">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>50</y>
      <width>261</width>
      <height>39</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Enter First Name</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_lname">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>100</y>
      <width>261</width>
      <height>41</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Enter Last Name</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_dob">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>220</y>
      <width>261</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>dd-mm-yy</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>270</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 16pt &quot;Trebuchet MS&quot;;</string>
    </property>
    <property name="text">
     <string>Email-id :</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_email">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>280</y>
      <width>261</width>
      <height>41</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>xyz@email.com</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_number">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>340</y>
      <width>261</width>
      <height>41</height>
     </rect>
    </property>
    <property name="maxLength">
     <number>10</number>
    </property>
    <property name="placeholderText">
     <string>+91..</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>210</y>
      <width>141</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 16pt &quot;Trebuchet MS&quot;;</string>
    </property>
    <property name="text">
     <string>Date of Birth :</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>130</y>
     <width>461</width>
     <height>471</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 20pt &quot;Franklin Gothic Heavy&quot;;</string>
   </property>
   <property name="title">
    <string>Sign-Up  Credentials</string>
   </property>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>291</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 16pt &quot;Trebuchet MS&quot;;</string>
    </property>
    <property name="text">
     <string>Enter your Desired Username :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>301</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 16pt &quot;Trebuchet MS&quot;;</string>
    </property>
    <property name="text">
     <string>Enter your Desired Password :</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_newuname">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>150</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>uname</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_newpwd">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>200</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="placeholderText">
     <string>pwd</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>250</y>
      <width>291</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 16pt &quot;Trebuchet MS&quot;;</string>
    </property>
    <property name="text">
     <string>Re-Enter your Password :</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_repwd">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>250</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="placeholderText">
     <string>repwd</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Validae">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>350</y>
      <width>261</width>
      <height>51</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(55, 167, 81);
border-radius:20%;


font: 75 16pt &quot;MS Shell Dlg 2&quot;;
font: 75 16pt &quot;Rockwell&quot;;</string>
    </property>
    <property name="text">
     <string>Validate (ID-Password)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>271</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 16pt &quot;Trebuchet MS&quot;;</string>
    </property>
    <property name="text">
     <string>Enter your ID-Card Number :</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_IdCard">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>100</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>id_no</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>10</y>
     <width>471</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Gadugi</family>
     <pointsize>26</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="text">
    <string>Student Registration Form</string>
   </property>
   <property name="textFormat">
    <enum>Qt::RichText</enum>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="img.qrc"/>
 </resources>
 <connections/>
</ui>
